src/App.tsx

```diff
@@ -1,52 +1,61 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { PageNavLink } from './components/PageNavLink/PageNavLink';
+import { TabsPage } from './Pages/TabsPage';
+import { HomePage } from './Pages/HomePage';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+const tabs = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
 
-export const App = () => (
-  <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
-    <nav
-      className="navbar is-light is-fixed-top is-mobile has-shadow"
-      data-cy="Nav"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
+export const App: React.FC = () => {
+  return (
+    <>
+      <nav
+        className="navbar is-light is-fixed-top is-mobile has-shadow"
+        data-cy="Nav"
+      >
+        <div className="container">
+          <div className="navbar-brand">
+            <PageNavLink to="/" text="Home" />
+            <PageNavLink to="/tabs" text="Tabs" />
+          </div>
         </div>
-      </div>
-    </nav>
+      </nav>
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route
+          path="/home"
+          element={<Navigate to="/" replace />}
+        />
+        <Route path="tabs">
+          <Route
+            index
+            element={(
+              <TabsPage
+                tabs={tabs}
+              />
+            )}
+          />
+          <Route
+            path=":tabId"
+            element={(
+              <TabsPage
+                tabs={tabs}
+              />
+            )}
+          />
+        </Route>
 
-    <div className="section">
-      <div className="container">
-        <h1 className="title">Home page</h1>
-        <h1 className="title">Tabs page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="tabs is-boxed">
-          <ul>
-            <li data-cy="Tab" className="is-active">
-              <a href="#/">Tab 1</a>
-            </li>
-            <li data-cy="Tab">
-              <a href="#/">Tab 2</a>
-            </li>
-            <li data-cy="Tab">
-              <a href="#/">Tab 3</a>
-            </li>
-          </ul>
-        </div>
-
-        <div className="block" data-cy="TabContent">
-          Please select a tab
-        </div>
-      </div>
-    </div>
-  </>
-);
+        <Route
+          path="*"
+          element={<h1 className="title">Page not found</h1>}
+        />
```

Comment:
Create a separate file for this, bc if you need to create 5 or more new routes, how big do you think this App component would be?
```suggestion
        <Route path="*" element={<NotFoundPage />} />
```